### ISSUE TITLE
[BUG] Video class has missing aria-label

### ISSUE THREAD

**Issue by pmirowski** [wet-boew/wet-boew#10189]
Error occurs with playwright accessibility testing: 
https://playwright.dev/docs/accessibility-testing

volume input results in an error.
https://wet-boew.github.io/v4.0-ci/demos/multimedia/multimedia-en.html

   Object {
     "description": "Ensure that every form element has a visible label and is not solely labeled using hidden labels, or the title or aria-describedby attributes",
     "help": "Form elements should have a visible label",
     "helpUrl": "https://dequeuniversity.com/rules/axe/4.11/label-title-only?application=playwright",
     "id": "label-title-only",
     "impact": "serious",
     "nodes": Array [
       Object {
         "all": Array [],
         "any": Array [],
         "failureSummary": "Fix all of the following:
   Only title used to generate label for form element",
         "html": "<input type=\"range\" class=\"volume\" aria-controls=\"wb-auto-3-md\" aria-labelledby=\"wb-auto-3-md\" aria-describedby=\"wb-auto-3-md-vlm-lbl\" title=\"Volume\" min=\"0\" max=\"100\" value=\"100\" step=\"5\">",
         "impact": "serious",
         "none": Array [
           Object {
             "data": null,
             "id": "title-only",
             "impact": "serious",
             "message": "Only title used to generate label for form element",
             "relatedNodes": Array [],
           },
         ],
         "target": Array [
           ".volume",
         ],
       },
     ],
     "tags": Array [
       "cat.forms",
       "best-practice",
     ],
   },

Please look into this
